src/components/table/makeSelectable.js

```diff
@@ -612,8 +608,27 @@ function makeSelectable(BaseTable) {
 
         isTargetSlider = event => event.target?.role === 'slider';
 
-        // Workaround for focus conflicting with Blueprint components in Search Quick Filters, not needed once Blueprint table is integrated
-        isQuickFilter = event => event.target?.dataset && 'radixCollectionItem' in event.target.dataset;
+        // Workaround for focus conflicting with Blueprint components for QuickSearch result, recent items and Quick Filters
+        isTargetQuickSearch = event => {
+            if (!event.target) {
+                return false;
+            }
+
+            const { className, dataset } = event.target;
+
+            if (
+                className &&
+                (className.includes('quickSearchRecentItem') || className.includes('quickSearchResultItem'))
```

Comment:
❓ optional chaining offer any value here? `className?.includes`

Reply:
Not sure it does since we are checking for className.

Reply:
I guess i mean dropping the `className` check and using optional chaining in both